### PR TITLE
[WIP] Comment submission page styles

### PR DESCRIPTION
**Pull request by xtine**
Set Comment submission styles for eregs/notice-and-comment#238

 - [x] Set alert styles
 - [x] PDF download link
 - [x] Comment tracking section
 - [x] What will happen to your comment list styles 

- [x] Comment Success page template
- [x] Comment Fail/Invalid page template

### CODE REVIEW

File: regulations/static/regulations/css/less/module/comment-confirm.less [hunk 1]
```diff
@@ -0,0 +1,42 @@
+
+.comment-confirm-success {
+  .font-regular;
+  padding: 30px 20px;
+  width: 80%;
+
+  .success-alert {
+    background-color: @green_light;
+    display: -webkit-flex;
+    display: flex;
```
Comment (cmc333333) [inline regulations/static/regulations/css/less/module/comment-confirm.less]: [Looks](http://caniuse.com/#feat=flexbox) like IE has significant issues with `display: flex`. Does it give us much?

File: regulations/templates/regulations/comment-confirm-success.html [hunk 2]
```diff
@@ -1,66 +1,81 @@
...
-    In most cases, EPA will post your comment to
-    <a href="#" target="_blank">the docket folder for this proposed rule</a>.
-    Then you will be able to find your comment there or by searching
-    <a href="https://www.regulations.gov" target="_blank">Regulations.gov</a>
-    for your comment tracking number or keywords from your comment.
-  </li>
-</ul>
+  <div class="save-pdf-copy">
+    <h3>Want to save a copy for yourself?</h3>
+    <a href="">Download your submitted comment (PDF)</a>
```
Comment (cmc333333) [inline regulations/templates/regulations/comment-confirm-success.html]: This isn't available on page render

File: regulations/static/regulations/css/less/main.less [hunk 3]
```diff
@@ -42,6 +42,7 @@ Modules
 @import "module/paragraph-markers.less";
 @import "module/comment.less";
 @import "module/comment-review.less";
+@import "module/comment-confirm.less";
```
Comment (cmc333333) [inline regulations/static/regulations/css/less/main.less]: :+1:

File: regulations/static/regulations/css/less/module/comment-confirm.less (hunk 1 above)
Comment (xtine) [inline regulations/static/regulations/css/less/module/comment-confirm.less]: I did a check in Virtualbox with IE11 and it loads okay. According to ATF analytics only 24% are on IE, and of that 87% are on IE11.

File: regulations/templates/regulations/comment-confirm-success.html (hunk 2 above)
Comment (xtine) [inline regulations/templates/regulations/comment-confirm-success.html]: Do you mean the PDF copy won't be available upon render or something else..?

Comment (cmc333333) [inline regulations/templates/regulations/comment-confirm-success.html]: The user won't be able to download the PDF upon page render; it's generated asynchronously and inserted via JS

File: regulations/static/regulations/css/less/module/comment-confirm.less (hunk 1 above)
Comment (cmc333333) [inline regulations/static/regulations/css/less/module/comment-confirm.less]: Over 3% of the total users are running IE below 11, and we don't know how indicative ATF is of our general users. I'm wondering if `flex` is worth it here; it seems like we can accomplish the same end goal without it pretty easily?

File: regulations/templates/regulations/comment-confirm-success.html (hunk 2 above)
Comment (xtine) [inline regulations/templates/regulations/comment-confirm-success.html]: Gotcha. I was just styling up the existing template so I'll keep this in mind.

**Comment by xtine** [comment]
Styled comment success page:
screen shot 2016-05-19 at 8 01 17 pm
screen shot 2016-05-19 at 8 01 53 pm

**Comment by xtine** [comment]
Styled comment submission fail page:
screen shot 2016-05-19 at 8 38 34 pm

**Comment by xtine** [comment]
Tasks remaining for another issue/PR:
- Download comment success PDF
- Comment success tracking number
- Link for "the docket folder for this proposed rule" and failed comment submission contact e-mail

File: regulations/templates/regulations/comment-confirm-fail.html [hunk 4]
```diff
@@ -1,20 +1,45 @@
...
-</ul>
+  <section class="problem-persist">
+    <h3>If this problem keeps happening</h3>
+    <p>
+      If you've already tried submitting your comment more than once, please save a copy of your draft comment for safekeeping.
+    </p>
+
+    <p>
+      <a href=""><span class="fa fa-file-pdf-o"></span> Download draft comment (PDF)</a>
+    </p>
```
Comment (cmc333333) [inline regulations/templates/regulations/comment-confirm-fail.html]: Let's comment this out until the JS functionality is available

File: regulations/templates/regulations/comment-confirm-invalid.html [hunk 5]
```diff
@@ -1,7 +1,45 @@
...
+
+  <section class="problem-persist">
+    <h3>If this problem keeps happening</h3>
+    <p>
+      If you've already tried submitting your comment more than once, please save a copy of your draft comment for safekeeping.
+    </p>
+
+    <p>
+      <a href=""><span class="fa fa-file-pdf-o"></span> Download draft comment (PDF)</a>
+    </p>
```
Comment (cmc333333) [inline regulations/templates/regulations/comment-confirm-invalid.html]: Ditto about commenting out.

File: regulations/templates/regulations/comment-confirm-success.html [hunk 6]
```diff
@@ -1,66 +1,87 @@
...
-    Then you will be able to find your comment there or by searching
-    <a href="https://www.regulations.gov" target="_blank">Regulations.gov</a>
-    for your comment tracking number or keywords from your comment.
-  </li>
-</ul>
+  <div class="save-pdf">
+    <div class="fa fa-file-pdf-o"></div>
+    <div class="save-pdf-text">
+      <h3>Want to save a copy for yourself?</h3>
+      <a href="">Download your submitted comment (PDF)</a>
```
Comment (cmc333333) [inline regulations/templates/regulations/comment-confirm-success.html]: This gets populated via JS. It shouldn't be present in the markup on load (hence "Waiting to receive PDF" in the original)

**Comment by cmc333333** [comment]
Without the comment success PDF and tracking number, this is a functionality regression. We shouldn't merge these styles until functionality is restored. Does that make sense?

**Comment by xtine** [comment]
I just commented out any mentions of PDF downloads. Looks like this:
screen shot 2016-05-19 at 11 06 26 pm

Can we merge styles like this and then add the PDF mentions once JS is in?

**Comment by cmc333333** [comment]
I don't think this can be merged with the JS regression; it'd block eregs/notice-and-comment#299 and eregs/notice-and-comment#306 . If you don't have time to try to fix, I can take a look later today

**Comment by cmc333333** [comment]
https://github.com/xtine/regulations-site/pull/4 brings back the JS functionality

**Comment by xtine** [comment]
@cmc333333: feel free to merge this, I'll push another PR for updates.

**Comment by cmc333333** [comment]
:+1:

**Pull request merged**